src/pybind/mgr/dashboard/frontend/src/app/core/auth/login/login.component.ts

```diff
@@ -56,11 +58,12 @@ export class LoginComponent implements OnInit {
         }
       });
     }
+    this.returnUrl = this.route.snapshot.queryParams['returnUrl'] || '/';
```

Comment:
Relocate to `login()` because the component can be created while the route snapshot contains an outdated value.
```suggestion
```

Reply:
Okay, thanks for pointing that out!